views/eApostilles/openEApp.ejs

```diff
@@ -0,0 +1,80 @@
+<% pageTitle="e-Apostille application" %>
+
+<div class="inner_header <%if (!user_data.loggedIn){%>no-user-signed-in<%}%>">
+    <a href="/dashboard" class="back-to-previous govuk-link">Back</a>
+    <%- partial ('../partials/inner-header.ejs')%>
+</div>
+<section class="column-three-quarters">
+    <span class="caption-xlarge">e-Apostille application</span>
+    <h1 class="heading-xlarge govuk-!-margin-top-2">
+        <%= applicationId %>
+    </h1>
+</section>
+<section class="column-two-thirds">
+    <p>
+        Your application has been successfully submitted.
+    </p>
+    <p>
+        We will notify you by email when it has been legalised. Your successfully legalised document will then
+        be
+        available for
+        download from this page.
+    </p>
+    <p>
+        You will have 21 days from when the documents have been successfully legalised to download them before
+        they are
+        deleted
+        for security reasons.
```

Comment:
what's up with the line breaks here? can it be just one or two lines? it's harder to read when it's all separated.  Is this using the application complete pattern too? https://design-system.service.gov.uk/patterns/confirmation-pages/

Reply:
Ha, I've honestly never noticed this. I copied and pasted it from Figma (I think), and this is how it came out. Good spot though.